cips/cip-19.md

```diff
@@ -311,6 +311,49 @@ Namespace data may span over multiple rows, in which case all the data is encaps
 containers. This enables parallelization of namespace data retrieval and certain [compositions](#protocol-compositions)
 may get advantage of that by requesting containers of a single namespace from multiple servers simultaneously.
 
+#### NamespaceRangeID
+
+NamespaceRangeID: encapsulates SampleID, Namespace and identifies the continuous range of shares in the DataSquare. Formated as below:
+
+```text
+NamespaceRangeID {
+  SampleID;
+  Namespace;
+  Length: u16;
+  OmitData: bool;	
```

Comment:
should OmitData also be present in `RowNamespaceId`?

Reply:
It may also be part of SampleID, i.e., one can request just the proof for the share/sample without the data itself. However, we are unaware of any use cases for the Row and Sample containers that can be requested without data. We could add that to be consistent with OmitData usage across types, but our approach is to specify and implement only what's known to be helpful in practice.